lib/features/home/presentation/widget/ideal/single_btn_select_dialog.dart

```diff
@@ -0,0 +1,156 @@
+import 'package:atwoz_app/app/constants/constants.dart';
+import 'package:atwoz_app/app/constants/enum.dart';
+import 'package:atwoz_app/app/widget/widget.dart';
+import 'package:atwoz_app/core/extension/extended_context.dart';
+import 'package:atwoz_app/features/home/presentation/controller/controller.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:go_router/go_router.dart';
+
+class SingleBtnSelectDialg extends ConsumerStatefulWidget {
+  const SingleBtnSelectDialg({
+    super.key,
+    required this.label,
+    required this.options,
+    required this.initialIndex,
+  });
+
+  final String label;
+  final List<String> options;
+  final int initialIndex;
+
+  @override
+  ConsumerState<SingleBtnSelectDialg> createState() =>
+      _SingleBtnSelectDialogState();
+}
+
+class _SingleBtnSelectDialogState extends ConsumerState<SingleBtnSelectDialg> {
+  late int _selectedIndex; // 초기 인덱스 중앙값으로 설정
+  late final FixedExtentScrollController _controller;
+
+  @override
+  void initState() {
+    super.initState();
+    _selectedIndex = widget.initialIndex;
+    _controller = FixedExtentScrollController(initialItem: _selectedIndex);
+  }
+
+  @override
+  void dispose() {
+    super.dispose();
+    _controller.dispose();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    final idealTypeNotifier = ref.watch(idealTypeNotifierProvider.notifier);
+    return Column(
+      mainAxisAlignment: MainAxisAlignment.center,
+      children: [
+        Container(
+          margin: EdgeInsets.symmetric(horizontal: 25),
+          padding: EdgeInsets.only(top: 16, bottom: 24),
+          width: context.screenWidth,
+          decoration: BoxDecoration(
+              color: Colors.white, borderRadius: BorderRadius.circular(8)),
```

Comment:
```suggestion
              color: Colors.white, 
              borderRadius: BorderRadius.circular(8),
          ),
```
가독성에 영향을 미치는 부분이다보니 컴마 사용을 자주 확인해주세요~